### DISCUSSION TITLE
ナビゲーションバーが表示されない

### DISCUSSION THREAD

**Discussion by 03ertuitui** [lit-iphone-course/lit-iphone-course#17] [Question-original] [answered]
todoアプリについてです！
viewcontrollerにてナビゲーションバーとテーブルビューを同じ一つの画面に配置しているのですが、シミュレーターを起動するとナビゲーションバーが消えてしまいます...
table
<img width="1512" alt="スクリーンショット 2022-09-07 12 40 02" src="https://user-images.githubusercontent.com/111737522/188783807-de2244d2-51ff-4359-8481-1e8ec6541a1c.png">
![Simulator Screen Shot - iPad Air (5th generation) - 2022-09-07 at 12 40 26](https://user-images.githubusercontent.com/111737522/188783815-22e5ec42-c7d2-4b82-a310-7f26fd4e2376.png)
二枚目はシミュレーターのスクリーンショットです。
GitHubのリンク↓
https://github.com/03ertuitui/Todo_1

**Comment by Yonpachi48** [comment]
commitが2日前のやつになっているので一度pushしてほしいです！！

**Reply by 03ertuitui** [reply]
すみませんでした！これでできますでしょうか...
https://github.com/03ertuitui/Todo_1

**Comment by Yonpachi48** [comment, marked as answer]
NavigationControllerを選択し、「Is Initial View Controller」にチェックを入れてみてください！
![スクリーンショット 2022-09-07 13 43 16](https://user-images.githubusercontent.com/76504399/188792366-d2793812-498e-4db5-9856-6d54b7bbf75a.png)

これはいわゆるentrypointのようなもので、これにチェックを入れた画面が一番最初に表示されます。

NavigationBarはNavigationControllerによってそれぞれの画面に紐付かれているため、階層のトップであるNavigationControllerが最初に開かれないと、表示されないということです！

**Reply by 03ertuitui** [reply]
できました！ありがとうございます！viewControllerの方にチェックを入れていたのですが、navigationcontrollerの方にチェックをつけておくべきだったのですね！！ありがとうございました！
